ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,201 @@
+
+### Review and Resolution
+
+The possible paths of the status of ZEPs are as follows:
+
+![Flowchart](/ZEP/assets/flowchart.png)
+
+All ZEPs should be created with the `Draft` status.
+
+Eventually, after the discussion, there may be a consensus that the ZEP should be accepted - see the next section for details. At this point, the status becomes 
+`Accepted`.
+
+Once a ZEP has been `Accepted`, the reference implementation must be completed. When the reference implementation is complete and incorporated into the main source 
+code repository, the status will be changed to `Final`.
+
+To allow gathering of additional design and interface feedback before committing to long term stability for a feature or standard library API, ZEP may also be 
+marked as `“Provisional”`. This is short for “Provisionally Accepted”, and indicates that the proposal has been accepted for inclusion in the reference 
+implementation, but additional user feedback is needed before the full design can be considered “Final”. Unlike regular accepted ZEPs, provisionally accepted 
+ZEPs may still be Rejected or Withdrawn even after the related changes have been included in a Zarr release.
+
+Wherever possible, it is considered preferable to reduce the scope of a proposal to avoid the need to rely on the `“Provisional”` status (e.g. by deferring some features to later ZEPs), as this status can lead to version compatibility challenges in the wider Zarr ecosystem.
+
+A ZEP can also be assigned status `Deferred`. The ZEP author or [Zarr Steering Council](https://github.com/zarr-developers/governance/blob/master/GOVERNANCE.md#steering-council) can assign the ZEP this status when no progress is being made on the ZEP.
+
+A ZEP can also be `Rejected`. Perhaps, after all, is said and done it was not a good idea. It is still important to have a record of this fact. The `Withdrawn` 
+status is similar—it means that the ZEP author themselves has decided that the ZEP is a bad idea, or has accepted that a competing proposal is a better alternative.
+
+When a ZEP is `Accepted`, `Rejected`, or `Withdrawn`, the ZEP should be updated accordingly. In addition to updating the status field, at the very least the 
+`Resolution` header should be added with a link to the relevant link of the discussion.
+
+ZEPs can also be `Superseded` by a different ZEP, rendering the original obsolete. The `Replaced-By` and `Replaces` headers should be added to the original and 
+new ZEPs respectively.
+Process ZEPs may also have a status of `Active` if they are never meant to be completed, e.g. ZEP 0 (this ZEP).
+
+### How does a ZEP becomes accepted?
+
+A ZEP is `Accepted` by the consensus of all interested contributors. We need a concrete way to tell whether consensus has been reached. When you think ZEP is ready to accept, create a new discussion in [zarr-python](https://github.com/zarr-developers/zarr-python/discussions) using ‘General’ as the category with a subject like:
```

Comment:
How are "interested contributors" defined?

Reply:
The term "interested contributors" here means all who were involved during the discussion of ZEP, users who will be affected by the change and the subset of the broader community who is interested in accepting/rejecting the change.

Reply:
see https://github.com/zarr-developers/governance/pull/17